src/piggui.rs

```diff
@@ -393,113 +480,37 @@ impl Application for Gpio {
+       |  | |  <version> | <hardware> | <status>                                          | | |
+       |  | +-----------------------------------------------------------------------------+ | |
+       |  +---------------------------------------------------------------------------------+ |
+       +--------------------------------------------------------------------------------------+
+    */
     fn view(&self) -> Element<Self::Message> {
-        let layout_selector = pick_list(
-            &Layout::ALL[..],
-            Some(self.chosen_layout),
-            Message::LayoutChanged,
-        )
-        .width(Length::Shrink)
-        .placeholder("Choose Layout");
-
-        let mut main_row = Row::new();
-
-        if let Some(hw_desc) = &self.hardware_description {
-            let layout_row = Row::new()
-                .push(layout_selector)
-                .align_items(Alignment::Start)
-                .spacing(10);
-
-            let hardware_desc_row = Row::new()
-                .push(hardware_view(hw_desc))
-                .align_items(Alignment::Center);
-
-            let file_button_style = ButtonStyle {
-                bg_color: Color::new(0.0, 1.0, 1.0, 1.0),
-                text_color: Color::BLACK,
-                hovered_bg_color: Color::new(0.0, 0.8, 0.8, 1.0),
-                hovered_text_color: Color::WHITE,
-                border_radius: 2.0,
-            };
-
-            let about_button_style = ButtonStyle {
-                bg_color: Color::TRANSPARENT,
-                text_color: Color::WHITE,
-                hovered_bg_color: Color::TRANSPARENT,
-                hovered_text_color: Color::new(0.7, 0.7, 0.7, 1.0),
-                border_radius: 4.0,
-            };
+        let main_col = Column::new().push(self.main_row()).push(info_row(self));
 
-            let mut configuration_column = Column::new().align_items(Alignment::Start).spacing(10);
-            configuration_column = configuration_column.push(layout_row);
-            configuration_column = configuration_column.push(hardware_desc_row);
-            configuration_column = configuration_column.push(
-                Button::new(Text::new("Save Configuration"))
-                    .style(file_button_style.get_button_style())
-                    .on_press(Message::Save),
-            );
-            configuration_column = configuration_column.push(
-                Button::new(Text::new("Load Configuration"))
-                    .style(file_button_style.get_button_style())
-                    .on_press(if !self.show_toast {
-                        // Add a new toast if `show_toast` is false
-                        Message::Load
-                    } else {
-                        // Close the existing toast if `show_toast` is true
-                        let index = self.toasts.len() - 1;
-                        Message::Toast(ToastMessage::Close(index))
-                    }),
-            );
+        let padding = Padding {
```

Comment:
Ig this is not necessary

Reply:
I did this so that status bar is closer to the bottom of the window, and no wasted space below it. "bottom" is 0.0

Reply:
I want to add a subtle line (Rule::) about it, maybe in a black color. But I didn't have time to see how to style it...